README.md

```diff
@@ -113,26 +113,13 @@ You can check out the examples by opening the html files in your browser, or che
 This object contains VuexFire internal mutations. They are all prefixed by
 `vuexfire/`. This object must be added in the root Store mutations object.
 
-### bindFirebaseRef(key, ref[, options])
+### bindFirebaseRef(key, ref)
 
 _Only available inside of an enhanced action_
 
 Binds a firebase reference to a property in the state. If there was already
 another reference bound to the same property, it unbinds it first.
```

Comment:
Could we add some explanation about it returning a promise that succeeds once the data is ready or that throws if anything goes wrong?

Reply:
Sure :) I updated the commit.

Reply:
PS: Quick question... I squashed locally and forced pushed my update – next time should I leave the commits separate and let you squash on your side?

Reply:
don't worry about it next time, I squash PRs 🙂